SPECS/python3-cassandra-driver/python3-cassandra-driver.spec

```
@@ -0,0 +1,84 @@
+Summary:        A modern, feature-rich and highly-tunable Python client library for Apache Cassandra (2.1+)
+Name:           python3-cassandra-driver
+Version:        3.24.0
+Release:        5%{?dist}
+Url:            https://github.com/datastax/python-driver#datastax-python-driver-for-apache-cassandra
+License:        ASL 2.0
+Group:          Development/Languages/Python
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Source0:        https://github.com/datastax/python-driver/archive/refs/tags/%{version}.tar.gz#/cassandra-driver-%{version}.tar.gz
+BuildRequires:  python3
+BuildRequires:  python3-Cython
+BuildRequires:  python3-libs
+BuildRequires:  python3-devel
```

`python3-devel` pulls in python3-libs and python3. We can simplify as such:

```suggestion
BuildRequires:  python3-Cython
BuildRequires:  python3-devel
```

---

Done